implementation/src/main/java/io/smallrye/config/ProfileConfigSourceInterceptor.java

```diff
@@ -0,0 +1,49 @@
+package io.smallrye.config;
+
+import java.util.Optional;
+
+import javax.annotation.Priority;
+
+@Priority(600)
+public class ProfileConfigSourceInterceptor implements ConfigSourceInterceptor {
+    public static final String SMALLRYE_PROFILE = "smallrye.config.profile";
+
+    private String profile;
+    private String config;
```

Comment:
I guess this is meant to be the configuration key, so its name probably ought to reflect that; also, since it is initialized in the constructor, it should be `final`.

Reply:
Oh yeah, this was from the old version.